docs/find-duplicate-skus/script.liquid

```diff
@@ -0,0 +1,103 @@
+{% assign mappings = hash %}
+{% assign dups = hash %}
+{% assign cursor = nil %}
+
+{% for i in (0..500) %}
+  {% assign after = i | times: 250 %}
+  {% capture query %}
+    query {
+      productVariants(
+        first: 250
+        after: {{ cursor | json }}
+      ) {
+        pageInfo {
+          hasNextPage
+        }
+        edges {
+          node {
+            id
+            sku
+            inventoryItem {
+              tracked
+            }
+          }
+          cursor
+        }
+      }
+    }
+  {% endcapture %}
+
+  {% assign result = query | shopify %}
+
+  {% if event.preview %}
+    {% capture result_json %}
+      {
+        "data": {
+          "productVariants": {
+            "edges": [
+              {
+                "node": {
+                  "id": "gid://shopify/ProductVariant/1234567890",
+                  "sku": "sku123",
+                  "inventoryItem": {
+                    "tracked": "true"
+                  }
+                }
+              },
+              {
+                "node": {
+                  "id": "gid://shopify/ProductVariant/123456789",
+                  "sku": "sku123",
+                  "inventoryItem": {
+                    "tracked": "true"
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    {% endcapture %}
+
+    {% assign result = result_json | parse_json %}
+  {% endif %}
+  
+  {% assign cursor = result.data.productVariants.edges.last.cursor %}
+  {% assign productVariants = result.data.productVariants.edges | map: "node" %}
+
+  {% for pv in productVariants %}
```

Comment:
This codebase goes for spelled-out, in general. I want to use either `variant` or `productVariant` here; no preference as to which. :)